### ISSUE TITLE
Session Expired Bug

### ISSUE THREAD

**Issue by aaronren** [heroiclabs/nakama-dart#90]
**Describe the bug**
Session always expired.

**Reproduction Code**
I found "expiresAt" and "refreshExpiresAt" is not parsed correct, 
in the function "Session.fromDto" in the file "session.dart".

```
expiresAt: DateTime.fromMillisecondsSinceEpoch(
  token['exp'] as int,
),
refreshExpiresAt: DateTime.fromMillisecondsSinceEpoch(
  refreshToken['exp'] as int,
),
```

It should be the same like codes in the function "Session.fromApi":
```
expiresAt: DateTime.fromMillisecondsSinceEpoch(
  (token['exp'] as int) * 1000,
),
refreshExpiresAt: DateTime.fromMillisecondsSinceEpoch(
  (refreshToken['exp'] as int) * 1000,
),
```

**Comment by meganide** [comment]
Also have this problem. Please fix asap.

**Comment by trey-a-hope** [comment]
Same problem here. I was wondering why my session was always expired after a fresh authentication.

**Comment by meganide** [comment]
> Same problem here. I was wondering why my session was always expired after a fresh authentication.

I saw that the fix has been merged in the next version release (1.0.6). I wonder when that will be release though.

**Comment by obrunsmann** [comment]
Release process started. Fixed from 1.0.6

You could use the release branch in the meantime if you want
